### DISCUSSION TITLE
Invoice By Square

### DISCUSSION THREAD

**Discussion by owlysk** [xseman/bysquare#19] [Q&A] [answered]
Ahoj,
v prvom rade ti chcem poďakovať za tvoju prácu. Takéto niečo malo vzniknúť už dávno a hlavne pôvodnými autormi "štandardu" ak chceme, aby QR kódy zjednodušovali život (aj vývojárom).
Neplánuješ náhodou robiť prevod aj voči Invoice bysquare?

**Comment by xseman** [comment]
Vďaka, cieľ je presne vytvoriť  dostupný štandard pre každého.

Čo sa týka Invoice, ten je naopak uzatvorený, čo vyžadovalo reverse engineering, na čo momentálne nemám dosť kapacity. No som otvorený spolupráci začať takýto projekt.

**Comment by owlysk** [comment]
No práve k Invoice som práve nevidel nikde žiadne špecifikácie ani informácie. Nemáš náhodou nejaké dokumenty k tomu Invoice bysquare? 
Otázka je, ak sa to ešte vo veľkom nepoužíva, či nevytvoriť a nepretlačiť nejaký iný open source a slobodný štandard pre faktúry komunitou, ktorý by bol dostupný úplne zdarma a pre každého.

**Comment by xseman** [comment, marked as answer]
@owlysk tak je to tu, oficiálne od verzie v4 podporujem aj invoice https://github.com/xseman/bysquare/pull/105
